README.md

```diff
@@ -82,8 +92,49 @@ Create an `index.html` file into `target/generated` directory and add the follow
 </html>
 ```
 
-Now run a web server locally inside the `target/generated` directory to see the `hello-triangle` in the browser.
-e.g. `python -m http.server`
+Now run a web server locally inside the `target/generated` directory to see the `hello-triangle` in the browser. Examples of servers are rust's 
+[`simple-http-server target/generated`](https://crates.io/crates/simple-http-server),
+[`miniserve target/generated`](https://crates.io/crates/miniserve),  or other languages 
+`ruby -run -e httpd -p 8000 target/generated`. 
+Python's debug server (`python -m http.server target/generated`) may have issues with MIME types causing failures. 
+
+##### WebGL
+
+First you need to copy the following files from spirv-cross's [wasm folder](https://github.com/grovesNL/spirv_cross/tree/master/wasm)
+and into your `target/generated`:
+
+- [`spirv_cross_wrapper_glsl.wasm`](https://github.com/grovesNL/spirv_cross/raw/master/wasm/spirv_cross_wrapper_glsl.wasm)
+- [`spirv_cross_wrapper_glsl.js`](https://github.com/grovesNL/spirv_cross/raw/master/wasm/spirv_cross_wrapper_glsl.js)
+
+Then, create an `index.html` file in `target/generated` directory and add the following code:
+
+```html
+<html>
```

Comment:
This seems to be quite verbose and technically dense. I don't think README of wgpu-rs is the right place to put this. Could we perhaps, move this into a wiki page that we link to?

Reply:
Agreed that a wiki or moving it under the examples (like gfx) would probably be better